_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,339 @@
+---
+layout:       pattern
+title:        Lists
+date:         2015-12-21 00:00:00
+
+categories:   patterns
+tags:
+- atomic
+- pattern
+- lists
+
+slug:         lists
+
+url_styles:     "patterns/_lists.scss"
+
+description: Lists are used to group related content in a number of settings and cases.
+
+info: Lists are used to group related content in a number of settings (e.g. ordered, unordered) and cases (e.g. definitions, links, etc.).
+---
+
+<h3 class="hd-6 example-set-hd">List (using different semantics)</h3>
+<div class="example-set">
```

Comment:
@talbs since these examples appear at the top of the "list" ;) they appear to be the preferred or default methods.  I would think that lists without visual item indicators (numbers for OL and bullets for UL) would be the exception and not the rule.  There are a lot of great examples below that would probably make better examples up here at the top.  That said, this entire section could benefit from introductory text that describes the various exceptions and typical use cases for those exceptions.